src/settings.rs

```diff
@@ -62,7 +62,7 @@ pub struct Settings {
     /// list of allowed vendors (Hash in JSON format)
     /// This consists of an advertiser name, and the associated filter settings
     /// (e.g. ```{"Example":{"advertizer_hosts":["example.com"."example.org"]}})```)
-    /// Unspecfied [AdmAdvertiserFilterSetttings] will use Default values specified
+    /// Unspecfied [crate::adm::AdmAdvertiserFilterSettings] will use Default values specified
```

Comment:
You'll need to prefix this block with `#[allow(rustdoc::private_intra_doc_links)]` to avoid a rustdoc warning.